core/src/main/java/org/apache/carbondata/core/util/AbstractDataFileFooterConverter.java

```diff
@@ -165,6 +165,7 @@ private static BitSet getPresenceMeta(
         dataFileFooter.setBlockInfo(new BlockInfo(tableBlockInfo));
         dataFileFooter.setSegmentInfo(segmentInfo);
         dataFileFooter.setVersionId(tableBlockInfo.getVersion());
+        dataFileFooter.setSchemaUpdatedTimeStamp(readIndexHeader.getSchema_time_stamp());
```

Comment:
What if it is from old store? better check `readIndexHeader.isSetSchema_time_stamp` before setting it to footer